OneSignalSDK/onesignal/build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
     api 'androidx.legacy:legacy-support-v4:[1.0.0, 1.99.99]'
     api 'androidx.browser:browser:[1.0.0, 1.99.99]'
     api 'androidx.appcompat:appcompat:[1.0.0, 1.99.99]'
+    api 'androidx.work:work-runtime:[2.0.0, 2.99.99]'
```

Comment:
Let's see if we can use 1.0.0. Ok if not, just better compatibility.

Reply:
`work-runtime` started at min `2.0.0`, we discussed we will keep range `[2.0.0, 2.99.99]`